server/game/cards/03-FF/Phantasm.js

```diff
@@ -0,0 +1,93 @@
+const GameActions = require('../../GameActions/index.js');
+const SpellCard = require('../../spellcard.js');
+
+class Phantasm extends SpellCard {
+    setupCardAbilities(ability) {
+        this.spellAction({
+            title: 'Noon: Phantasm: Move Unbooted Dude',
```

Comment:
I do not think we need name of the card in title as it is displayed in the prompt anyway.